code/game/objects/structures.dm

```diff
@@ -64,6 +64,11 @@
 		step(O, get_dir(O, src))
 	return
 
+/obj/structure/proc/do_climb(atom/movable/A)
+	density = 0

```

Comment:
Are there any climbable objects with a density of 0? You should at least account for them